src/lib/isEmail.js

```diff
@@ -141,11 +141,25 @@ export default function isEmail(str, options) {
     }
   }
 
+  /**
+  * Check the first or last string it ends with special characters.
+  * Also, check if special characters apear consecutively.
+  */
+  const regEndsWithSpecialChar = /^.*[!#$%&',*+-\/=?^_`{|]$/i;
+  const regStartsWithSpecialChar = /^[!#$%&',*+-\/=?^_`{|]/i;
+  const regConsecutiveSpecialChar = /([!#$%&',*+-\/=?^_`{|])\1/i;
```

Comment:
@profnandaa is there a way to make this regex validate abc_+kl@gmail.com. For now, it validates abc__dc@mail.com.
In short, what I mean is that I have caught most of the case of invalid emails, except an instance where when two instances of special characters that are different e.g -+ occur concurrently.
See example above.

Reply:
Not sure if i'm understanding correctly or not the use case but is ```/([!#$%&',*+-\/=?^_`{|]){2,}/``` fixing your problem? BTW, we don't need the case insensitive flag since we are only searching for special characters